### ISSUE TITLE
I don't know   

### ISSUE THREAD

**Issue by SarahGauth** [spyder-ide/spyder#13889]
### What steps will reproduce the problem?

I do not know exacly what appened. This message or error was displayed when I used the ''tab'' on my keyboard and when I tried to open a project file. 

Thank you. 

### Traceback
```python-traceback
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\projects\plugin.py", line 98, in <lambda>
    triggered=lambda v: self.open_project())
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\projects\plugin.py", line 288, in open_project
    self.notify_project_open(path)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\completion\decorators.py", line 22, in wrapper
    self.emit_request(method, params, requires_response)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\projects\plugin.py", line 553, in emit_request
    self.main.completions.broadcast_notification(method, params)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\completion\plugin.py", line 322, in broadcast_notification
    client_info['plugin'].broadcast_notification(
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\completion\languageserver\plugin.py", line 578, in broadcast_notification
    self.send_notification(language, request, params)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\completion\languageserver\plugin.py", line 569, in send_notification
    client.perform_request(request, params)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\completion\languageserver\client.py", line 527, in perform_request
    _id = handler(params)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\completion\languageserver\decorators.py", line 41, in wrapper
    params = req(self, *args, **kwargs)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\completion\languageserver\providers\workspace.py", line 46, in send_workspace_folders_change
    folder_uri = path_as_uri(folder)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\completion\languageserver\providers\utils.py", line 51, in path_as_uri
    path_obj = pathlib.Path(osp.abspath(path))
  File "C:\ProgramData\Anaconda3\lib\ntpath.py", line 527, in abspath
    return normpath(_getfullpathname(path))
TypeError: _getfullpathname: path should be string, bytes or os.PathLike, not NoneType
```

## Versions

* Spyder version: 4.1.4 
* Python version: 3.8.3
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.5.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.16.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=1.0                   :  2.5.3 (OK)
pyls >=0.34.0;<1.0.0           :  0.34.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.5 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.1.2 (OK)
spyder_kernels >=1.9.2;<1.10.0 :  1.9.2 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.1 (OK)

# Optional:
cython >=0.21                  :  0.29.21 (OK)
matplotlib >=2.0.0             :  3.2.2 (OK)
numpy >=1.7                    :  1.18.5 (OK)
pandas >=0.13.1                :  1.0.5 (OK)
scipy >=0.17.0                 :  1.5.0 (OK)
sympy >=0.7.3                  :  1.6.1 (OK)
```

**Comment by SarahGauth** [comment]
Good to know! Thank you and have a nice day!

**Comment by ccordoba12** [comment]
This is fixed in our 4.1.5 version, please update.

**Comment by SarahGauth** [comment]
I thought I had downloaded the most recent version, but apparently not. Thank you for the follow up!

Sarah Gauthier
________________________________
Candidate au doctorat en sciences géographiques
Département de géographie
Centre d'études nordiques
Université Laval, Québec (Qc)
________________________________
De : Carlos Cordoba <notifications@github.com>
Envoyé : mercredi 30 septembre 2020 09:03
À : spyder-ide/spyder <spyder@noreply.github.com>
Cc : Sarah Gauthier <sarah.gauthier.1@ulaval.ca>; Author <author@noreply.github.com>
Objet : Re: [spyder-ide/spyder] I don't know (#13889)

[Externe UL*]

This is fixed in our 4.1.5 version, please update.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fspyder-ide%2Fspyder%2Fissues%2F13889%23issuecomment-701376249&data=02%7C01%7Csarah.gauthier.1%40ulaval.ca%7C64f0f2c46e554453522b08d865414122%7C56778bd56a3f4bd3a26593163e4d5bfe%7C1%7C0%7C637370678220259761&sdata=97BGMtMcrnXDf5VGOKT4kQgRo4OMoX3Qz04S4FWNhi0%3D&reserved=0>, or unsubscribe<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FARGLTQ46W7IWLMQZI2TL46LSIMUALANCNFSM4R67KOLA&data=02%7C01%7Csarah.gauthier.1%40ulaval.ca%7C64f0f2c46e554453522b08d865414122%7C56778bd56a3f4bd3a26593163e4d5bfe%7C1%7C0%7C637370678220269757&sdata=ReOWb6a3n3OgUqauKAR3vKdn9DmBLYy6OTIZrtLXVRU%3D&reserved=0>.

*ATTENTION : L’émetteur de ce courriel est externe à l’Université Laval.
Évitez de cliquer sur un hyperlien, d’ouvrir une pièce jointe ou de transmettre des informations si vous ne connaissez pas l’expéditeur du courriel. En cas de doute, contactez l’équipe de soutien informatique de votre unité ou hameconnage@ulaval.ca.
________________________________

**Comment by ccordoba12** [comment]
Sometimes Anaconda doesn't come with the latest version of Spyder. Please update by closing Spyder, opening the Anaconda Prompt and running there

    conda update anaconda 
    conda install spyder=4.1.5
